borg/archiver.py

```diff
@@ -725,6 +725,15 @@ def write(bytestring):
     @with_archive
     def do_info(self, args, repository, manifest, key, archive, cache):
         """Show archive details such as disk space used"""
+        def format_argument(argument):
+            if ' ' in argument:
+                return '"{}"'.format(argument)
+            else:
+                return argument
```

Comment:
[`shlex.quote()`](https://docs.python.org/3/library/shlex.html#shlex.quote) should be more "solid"